pyproject.toml

```diff
@@ -62,6 +62,7 @@ doc = [
 	"flake8-docstrings",
 	"black",
 	"furo",
+  "isort",
```

Comment:
It looks like flake8 didn't catch the correct indentation on this 😅

Reply:
wait... Does flake8 cover toml files as well?
If not I can at least add a pre-commit hook to help "enforce" some formatting of such support / config files (yaml, json, toml).
There are a couple of options available.
https://pre-commit.com/hooks.html

Reply:
turned out this was a double space instead of a tab...